### PR TITLE
UXENG-2239 Add paragraph about publishing SCM algos

### PR DESCRIPTION
**Pull request by bnlb**
Resolves [UXENG-2239](https://algorithmia.atlassian.net/browse/UXENG-2239)

### CODE REVIEW

File: _pages/algorithm-development/source-control-management.md [hunk 1]
```diff
@@ -136,6 +136,18 @@ Once you've created your algorithm, any commits to your GitHub repository's _def
...
+
+When you're ready to create a new version of your algorithm, visit that algorithm's page and select the __Builds__ tab. Find the Git hash of the build you'd like to use, and then click __Publish__ on the right side of the row.
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/source_code_management/builds_page.png" alt="Algorithm build page" class="screenshot img-sm">
+
+This will open a publish modal that will let you publish your algorithm.
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/source_code_management/publish_modal.png" alt="Publish algorithm modal" class="screenshot img-sm">
+
+If you don't see any builds listed, make sure that you're pushing commits to your repository's _default_ branch. Builds are only triggered for commits pushed to the default branch.
```
Comment (ryanmillerdev) [inline _pages/algorithm-development/source-control-management.md:149]: I'd add a line after this paragraph with something like, "To learn more about algorithm builds, [click here](https://algorithmia.com/developers/platform/algorithm-profiles#algorithm-builds)."

File: _pages/algorithm-development/source-control-management.md [hunk 2]
```diff
@@ -136,6 +136,18 @@ Once you've created your algorithm, any commits to your GitHub repository's _def
 
 You can view your algorithm's builds by heading to it's landing page and clicking the "Builds" tab if you are the algorithm's owner.
 
+#### Publishing GitHub-Hosted Algorithms
+
+When you're ready to create a new version of your algorithm, visit that algorithm's page and select the __Builds__ tab. Find the Git hash of the build you'd like to use, and then click __Publish__ on the right side of the row.
```
Comment (ryanmillerdev) [inline _pages/algorithm-development/source-control-management.md]: :bulb: Might tweak this phrasing to "Locate the build corresponding to specific commit SHA you wish to use, and then click **Publish** on the right side of the row."

**Review by ryanmillerdev — APPROVED** [review]
Thanks for adding this! 📖

File: _pages/algorithm-development/source-control-management.md (hunk 1 above)
Comment (bnlb) [reply _pages/algorithm-development/source-control-management.md:149]: Good call, updating.

**Pull request merged**